cobalt/android/apk/app/src/main/java/dev/cobalt/media/MediaCodecBridge.java

```diff
@@ -659,6 +659,9 @@ private void updateOperatingRate() {
   private int flush() {
     try {
       mMediaCodec.get().flush();
+      if (mFrameRateEstimator != null) {
+        mFrameRateEstimator.reset();
```

Comment:
I'd move these three new lines to outside the try-catch, so that
it's clear what can throw (maybe add a `finally` and put the
new code there).